kis_api.py

```
@@ -1,372 +1,850 @@
+                "CTX_AREA_FK200": "",
+                "CTX_AREA_NK200": ""
+            }
+
+            # API 호출
+            response = requests.get(url, headers=headers, params=params, timeout=10)
+            balance = response.json()
+
+            if balance and balance.get('rt_cd') == '0':
+                output1 = balance.get('output1', [])
+                output2 = balance.get('output2', [])
+                output3 = balance.get('output3', {})
```

Ruff (0.14.0)

Local variable `output3` is assigned to but never used

Remove assignment to unused variable `output3`

(F841)